libs/keycloak-config/src/keycloak-config.service.ts

```
@@ -0,0 +1,174 @@
+    const clients = await this.commonService.httpGet(clientsUrl, this.getAuthHeader(token));
+
+    if (!Array.isArray(clients)) {
+      this.logger.warn('No clients found in realm');
+      return;
+    }
+
+    this.logger.log(`Found ${clients.length} total clients in realm`);
+
+    const orgClients = clients.filter(
+      (client: { serviceAccountsEnabled: boolean; clientId: string }) =>
+        client.serviceAccountsEnabled && !client.clientId.startsWith('realm-')
```

ESLint

[error] Expected no linebreak before this expression.

(implicit-arrow-linebreak)